components/Grid/Template/TemplateGrid.vue

```diff
@@ -6,17 +6,31 @@
     <div class="template-designer">
         <div class="horizontal-wrapper">
             <VerticalTabBar :items="verticalTabs" />
-            <div class="grid-wrapper">
-                <div class="template-designer__grid">
-                    <template
-                        v-for="(layoutElement, index) in layout">
-                        <Component
-                            :is="getComponentViaName(layoutElement.component)"
-                            :key="index"
-                            v-bind="{ layoutElement }" />
+            <TemplateGridDesigner @rowsCount="onRowsCountChange">
+                <TemplateGridPresentationLayer
+                    :grid-styles="gridStyles"
+                    :columns="columnsNumber"
+                    :rows="maxRows"
+                    :is-header="false" />
+                <div
+                    :style="gridStyles"
+                    class="draggable-grid">
+                    <template v-for="item in draggableGridAreaLenght">
+                        <TemplateGridGhostItem
+                            :key="item"
+                            :position="{
+                                row: Math.ceil(item / columnsNumber),
```

Comment:
consider to put 'row' and 'column' expression to function with proper name in component

Reply:
I did it in next PR 💃